its/plugin/tests/src/test/java/com/sonar/it/php/CommonRulesTest.java

```diff
@@ -1,97 +0,0 @@
-/*
- * SonarQube PHP Plugin
- * Copyright (C) 2011-2016 SonarSource SA
- * mailto:contact AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-package com.sonar.it.php;
-
-import com.google.common.base.Charsets;
-import com.google.common.io.Files;
-import com.sonar.orchestrator.Orchestrator;
-import com.sonar.orchestrator.build.SonarRunner;
-import java.io.File;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
-import org.junit.Test;
-import org.sonar.wsclient.issue.IssueQuery;
-
-import static org.fest.assertions.Assertions.assertThat;
-
-public class CommonRulesTest {
```

Comment:
@ivandalbosco why test is removed in this commit? I would keep it, nothing wrong in testing that common rules are provided by platform.